src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -73,7 +74,7 @@ class LlmDropdown(
     val selectedFromState = chatModelProviderFromState
     val selectedFromHistory = HistoryService.getInstance(project).getDefaultLlm()
     val selectedModel =
```

Comment:
Double check if we need to handle nulls (during init)

Reply:
I wanted to add additional check but `it.model` cannot be null so IDE was saying that check was redundant.